TextRPG_Team25/Battle.cs

```diff
@@ -1,7 +1,42 @@
-﻿namespace TextRPG_Team25
+﻿using System.Numerics;
+using System.Threading;
+
+namespace TextRPG_Team25
 {
     internal class Battle
     {
+        public void MonsterAttack()
+        {
+            int battleHp = player.hp; //전투에서쓰는 HP따로 선언
+
+
+        for (int battleEnermy = 0; battleEnermy <= EnermyList.Count; battleEnermy++)  //랜덤으로 생성된 적 개체수만큼 실행
+            {
+
+                int randomDmg = new Random().Next(1, 11) //적 공격력의 오차값 10퍼센트
```

Comment:
적 공격력의 10%가 오차값이기 때문에 따로 설정해야 합니다..! 제가 플레이어 공격 구현할 때에는 아래와 같이 구현했습니다
```suggestion
                int offset = (int)Math.Ceiling(baseAttack * 0.1f); // Math.Celiling: 올림 함수
                int damage = new Random().Next(baseAttack - offset, baseAttack + offset + 1);
```